code/drasil-printers/Language/Drasil/TeX/Print.hs

```diff
@@ -333,7 +333,7 @@ makeDefTable :: PrintingInformation -> [(String,[LayoutObj])] -> D -> D
 makeDefTable _ [] _ = error "Trying to make empty Data Defn"
 makeDefTable sm ps l = vcat [
   pure $ text 
-  $ "\\begin{tabular}{p{"++show colAwidth++"\\textwidth} p{"++show colBwidth++"\\textwidth}}",
+  $ "\\begin{tabular}{>{\\raggedright}p{"++show colAwidth++"\\textwidth}>{\\raggedright\\arraybackslash}p{"++show colBwidth++"\\textwidth}}",
```

Comment:
hard-coded widths like this seem like a bad idea? This doesn't feel future-proof.

Reply:
Oh, I see that these are percentages of \textwidth - that's not too bad then.